### PR TITLE
[net7-main] Fixes to get solution building in VS

### PR DESCRIPTION
**Pull request by CodeBlanch**
These fixes got main building in VS with the recent net7 changes. Works on my machine™

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
     "rollForward": "latestFeature",
-    "version": "6.0.100",
-    "allowPrerelease": "true"
+    "version": "7.0.100-preview",
```
Comment (cijothomas) [inline global.json:4]: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md#development-environment we need to update this to include install .net 7 preview?

**Comment by alanwest** [comment]
Integration tests failing to build now?

File: global.json (hunk 1 above)
Comment (CodeBlanch) [reply global.json]: Updated

File: global.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
   "sdk": {
     "rollForward": "latestFeature",
-    "version": "6.0.100",
-    "allowPrerelease": "true"
+    "version": "7.0.100-preview"
```
Comment (vishweshbankwar) [inline global.json:4]: we need exact version here?
https://docs.microsoft.com/en-us/dotnet/core/tools/global-json#version

Comment (CodeBlanch) [reply global.json:4]: I tried both. Seems to work fine with just "-preview" so I went with the less restrictive version.

File: .github/workflows/integration-md.yml [hunk 3]
```diff
@@ -18,22 +18,22 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: [net6.0]
+        version: [net7.0]
```
Comment (alanwest) [inline .github/workflows/integration-md.yml:21]: My gut says this is fine. In the before world, there were more significant differences between versions of .NET. The OTLP exporter integration test exercised the exporter using the two different gRPC libraries. This difference does not exist between net6 and net7.

**Review by alanwest — APPROVED** [review]

File: .github/workflows/integration-md.yml (hunk 3 above)
Comment (alanwest) [reply .github/workflows/integration-md.yml:21]: Also +1 to keeping the CI short!

Comment (CodeBlanch) [reply .github/workflows/integration-md.yml:21]: We can look at putting it back. My main goal right now is just to unblock VS.

**Pull request merged**